### PR TITLE
add acs style

### PR DESCRIPTION
**Pull request by yufree**
I have tested acs articles on both mac 10.11 and windows 7, it all passed the check.

This is a clean commit with 12 files changed. Pls close last dirty one. https://github.com/rstudio/rticles/pull/57 @jjallaire 

However, the elsevier_article can't pass the check on windows 7 as shown below. I think it may be caused by update of pandoc recently.

```
running command '"C:/Users/yufree/AppData/Local/Pandoc/pandoc" +RTS -K512m -RTS testdoc.utf8.md --to latex --from markdown+ascii_identifiers+tex_math_single_backslash-implicit_figures-autolink_bare_uris --output "C:\Users\yufree\AppData\Local\Temp\RtmpQXxJBD\file5dc252c2f7b.pdf" --filter pandoc-citeproc --template "C:\Users\yufree\AppData\Local\Temp\RtmpErFleb\RLIBS_d4051c42b45\rticles\rmarkdown\templates\elsevier_article\resources\template.tex" --highlight-style tango --latex-engine pdflatex --bibliography mybibfile.bib' had status 43 
```

### CODE REVIEW

**Comment by yufree** [comment]
@jjallaire @yihui Thanks a lot for your suggestions! 

I have revised the code as you said. Again it passed the tests on mac and windows.

**Comment by jjallaire** [comment]
When I run the test on the Mac I get this error:

```
! Undefined control sequence.
l.107 ...e entries correctly. Notice that ``\latin
```

File: R/acs_article.R [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+acs_article <- function(...,
+                        keep_tex = TRUE,
+                        md_extensions = c("-autolink_bare_uris")){
+  pdf_document_format(...,
+                      keep_tex = keep_tex,
+                      md_extensions = md_extensions,
+                      format = "acs_article",
+                      template = "template.tex",
+                      csl = "american-chemical-society.csl",
+                      fig_caption = TRUE)
```
Comment (jjallaire) [inline R/acs_article.R]: You should include `fig_caption` in the signature as well (just like `keep_tex`).

**Comment by jjallaire** [comment]
When re-testing this I initially got errors on Windows but then updated my
version of MikTeX and those errors went away. I can now successfully knit
in RStudio however when I run the package tests I get this error:

! Missing $ inserted.
<inserted text>
                $

I don't know what to make of this and don't see why this should hold up
taking the PR. If you have any insight let me know, otherwise as I said
I'll take the PR anyway since it works from within RStudio.

File: inst/rmarkdown/templates/acs_article/template.yaml [hunk 2]
```diff
@@ -0,0 +1,4 @@
+name: ACS
```
Comment (jjallaire) [inline inst/rmarkdown/templates/acs_article/template.yaml]: You should spell out American Chemical Society here rather than using the abbreviation

**Comment by yufree** [comment]
Thanks! @jjallaire 

The `Undefined control sequence` and `Missing $` issues can't be reproduced in my computer both on mac and windows. I think the first one might come from old bugs of the latex package `achemso`. The second one I also have no idea...

Other issues have been revised and pass the check.

**Comment by jjallaire** [comment]
I updated to MacTex 2015 and the Mac problems went away! Will merge shortly.

**Pull request merged**